wallaby/app/assets/stylesheets/wallaby/base/_override_vars.scss

```diff
@@ -0,0 +1,3 @@
+// Override the vars for the `core.scss`
+$background-color: #f3f6fb;
```

Comment:
These two variables are set in `core.scss` at https://github.com/wallaby-rails/wallaby-rails/blob/fix-vars/wallaby/app/assets/stylesheets/wallaby/core.scss#L11121-L11122 by summer_note. We will need to override it for the rest of the stylesheets to work properly.